components/post/BestPost.tsx

```diff
@@ -0,0 +1,40 @@
+import formatDate from '@/utils/formatDate';
+import Image from 'next/image';
+import BestBadge from './BestBadge';
+import PostImage from './PostImage';
+import styles from './Post.module.scss';
+
+function BestPost({ article }: { article: IPost }) {
+  const { title, content, image, writer, likeCount, createdAt } = article;
```

Comment:
```suggestion
function BestPost({ article: {title, content, image, writer, likeCount, createdAt} }: { article: IPost }) {
```

article이 따로 안쓰여서 이렇게 바로 구조분해할당 할 수도 있습니다!